cmd/publishing-bot/config/rules.go

```diff
@@ -46,7 +47,7 @@ func (c Source) String() string {
 
 type BranchRule struct {
 	Name string `yaml:"name"`
-	// a (full) version string like 1.10.2.
+	// a (full) semver version string like 1.10.2.
```

Comment:
so 1.10 works too?

Reply:
Updated so that 1.10 works but 1.10.0 won't. PTAL.

Reply:
I don't really like this hack though...

Reply:
Actually there is no non-full semver version string. It's pretty clearly defined in https://semver.org/.

Reply:
so which do we want? Semver or not semver? :-)

Reply:
I updated it to say "valid Go version" :)

https://github.com/kubernetes/publishing-bot/pull/232/files#diff-d058080bd6d5b27f3f6ff87d286765e8R50